core/metacling/src/TCling.cxx

```diff
@@ -5894,6 +5898,8 @@ static void* LazyFunctionCreatorAutoloadForModule(const std::string& mangled_nam
       const HeaderSearchOptions &HSOpts = PP.getHeaderSearchInfo().getHeaderSearchOpts();
       const auto ModulePaths(HSOpts.PrebuiltModulePaths);
       cling::DynamicLibraryManager* dyLibManager = fInterpreter->getDynamicLibraryManager();
+      // get separator like "/"
+      StringRef separator = get_separator();
```

Comment:
I'd like to see the full namespace qualification (`llvm::..::get_separator()`). This makes the code more readable.